orion/pkg/router/handler.go

```diff
@@ -29,13 +29,14 @@ func (h *Handler) SetupApiEndpoints() {
 		apiClasses.GET("/v1/class/:classId", controllers.GetClassById)
 		apiClasses.POST("/v1/class/:classId", controllers.UpdateClass)
 		apiClasses.DELETE("/v1/class/:classId", controllers.DeleteClass)
+		apiClasses.GET("/v1/class/:classId/sessions", controllers.GetAllSessionsByClassId)
```

Comment:
Sorry again. I looked at this again and thought it was confusing.
Can you change this back to `/v1/sessions/class/:classId`
It should represent "getting a bunch of sessions with the class `:classId`"

And place it after the DELETE sessions endpoint. We want to keep the X by Id endpoints together for readability.